src/System/ref/Compat/System.cs

```diff
@@ -4995,14 +4995,14 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public CookieCollection() { }
         public int Count { get { return default(int); } }
         public bool IsReadOnly { get { return default(bool); } }
-        public bool IsSynchronized { get { return default(bool); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
+        object System.Collections.ICollection.SyncRoot { get { return default(object); } }
         public System.Net.Cookie this[int index] { get { return default(System.Net.Cookie); } }
         public System.Net.Cookie this[string name] { get { return default(System.Net.Cookie); } }
-        public object SyncRoot { get { return default(object); } }
         public void Add(System.Net.Cookie cookie) { }
         public void Add(System.Net.CookieCollection cookies) { }
         public void CopyTo(System.Array array, int index) { }
-        public void CopyTo(System.Net.Cookie[] array, int index) { }
+        void System.Collections.ICollection.CopyTo(System.Array array, int index) { }
```

Comment:
This is the way the reference assembly for the [actual implementation](https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L56) is. Implementation doesn't have a definition for the `Cookie[]` overload, which is why I moved it to mimic the current implementation.